test/test_module/future_div.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import division
-import torch
+import torch  # noqa: F401
```

Comment:
@zou3519 I assumed that the `import torch` here was intentional

Reply:
yes, although nothing bad will happen if we delete it. This file is supposed to emulate some code that a user would write